### PR TITLE
Added UnsupportedDatabaseTypeException

### PR DESCRIPTION
**Pull request by smilesrg**

### CODE REVIEW

File: src/Exception/UnsupportedDatabaseTypeException.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Port\Exception;
+
+use Port\Exception;
+
+class UnsupportedDatabaseTypeException extends \Exception implements Exception
+{
+    /**
+     * @param array $duplicates
```
Comment (ddeboer) [inline src/Exception/UnsupportedDatabaseTypeException.php]: Typehint is wrong.

**Comment by sagikazarmark** [comment]
Shouldn't this one go to the relevant package instead of the main one?

**Comment by smilesrg** [comment]
@sagikazarmark don't know. Probably not.

**Comment by sagikazarmark** [comment]
Why not? It has nothing to do with the core package IMO.

**Comment by smilesrg** [comment]
Okay, I'll put it to https://github.com/portphp/doctrine-adapter/

**Comment by smilesrg** [comment]
you can close this PR then

**Pull request closed without merging**